mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -39,7 +43,10 @@ def _validate_tracking_uri_is_databricks(scorer_name: str) -> None:
         )
 
 
-class BuiltInScorer(Scorer):
+from mlflow.genai.judges.base import Judge
+
+
+class BuiltInScorer(Judge):
```

Comment:
In the future, we may want to add built-in scorers that aren't AI judges, e.g. scorers that rely purely on deterministic Python code.

Perhaps we should add a `BuiltinJudge` class that inherits from `BuiltInScorer` and `Judge`?